groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
```

Comment:
The methods `create_group_category()`,  `api_handler()`, and possibly others should be part of a class in a module separate from `groupsforsections.py`.  They will be useful in other projects, too.  (For example, possibly the ArcGIS integration which I am starting.)  So, perhaps they belong in a separate repository so that they may be included in other Python projects using `pip install`.

For that matter, are there any packages already available for interacting with JSON-based web service APIs?